src/content/en/_shared/helpful.html

```diff
@@ -0,0 +1,38 @@
+<style>
+  .helpful--container {
+    margin: 1em 0;
+    position: relative;
+  }
+  .helpful--section {
+    position: static;
+  }
+</style>
+
+<p>Was this page helpful?</p>
+
+<div class="helpful--container">
+  <section class="expandable helpful--section">
+    <button class="button button-primary expand-control gc-analytics-event"
+            data-category="Helpful" data-value="1"
```

Comment:
Both of these (Yes/No) have the same data-value, was that intentional?

Reply:
Not intentional. Thank you for the review.